critiquebrainz/db/moderation_log.py

```diff
@@ -0,0 +1,139 @@
+"""
+Methods here define logs for various activities that the moderators can take
+via the moderator interface. A new log entry is created for every action.
+"""
+from critiquebrainz.data import db
+import sqlalchemy
+from datetime import datetime
+
+
+ACTION_HIDE_REVIEW = "hide_review"
+ACTION_BLOCK_USER = "block_user"
+
+def create(*, admin_id, review_id=None, user_id=None,
+           action, reason):
+    """Log blocking a user or hiding a review with IDs.
```

Comment:
> blocking a user or hiding a review

This doesn't have to be so specific. You could just say something like "Make a record in the moderation log."